README.md

```diff
@@ -2,9 +2,31 @@
 
 ---
 
-[![PyPI version](https://img.shields.io/pypi/v/selectolax.svg)](https://pypi.python.org/pypi/selectolax)
+An extremely fast HTML5 parser with CSS selectors, written in Cython,
+using [Modest](https://github.com/lexborisov/Modest/) and [Lexbor](https://github.com/lexbor/lexbor) engines!
 
-A fast HTML5 parser with CSS selectors using [Modest](https://github.com/lexborisov/Modest/) and [Lexbor](https://github.com/lexbor/lexbor) engines.
+---
+
+[![PyPI - Version](https://img.shields.io/pypi/v/selectolax?logo=pypi&label=Pypi&logoColor=fff)](https://pypi.org/project/selectolax)
```

Comment:
That looks like a Christmas tree now.

Let's keep: current PyPI version, license, total downloads, and supported versions, CI status.

Displaying too much information does not benefit users.